ci/cd-manager.cue

```diff
@@ -0,0 +1,89 @@
+package ci
+
+import (
+	"dagger.io/dagger"
+
+	"universe.dagger.io/aws"
+	"universe.dagger.io/docker"
+
+	"github.com/3box/pipeline-tools/utils"
+)
+
+#Branch: "develop" | "release-candidate" | "main"
+#EnvTag: "dev" | "qa" | "tnet" | "prod"
+#Sha:	 =~"[0-9a-f]{40}"
+#ShaTag: =~"[0-9a-f]{12}"
+
+dagger.#Plan & {
+	client: env: {
+		// Secrets
+		AWS_ACCOUNT_ID:        string
+		AWS_REGION:    		   string
+		AWS_ACCESS_KEY_ID:     dagger.#Secret
```

Comment:
does #Secret do something or is this a TODO?

Reply:
It represents a secret in Dagger, which then masks it in all the logs.